src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsytem.java

```diff
@@ -8,7 +8,9 @@
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import io.github.oblarg.oblog.Loggable;
 
-public class DrivebaseSubsystem extends SubsystemBase implements Loggable{
+import frc.team2412.robot.commands.DriveCommand;
+
+public class DrivebaseSubsytem extends SubsystemBase implements Loggable{
```

Comment:
Please rename the file & class to fix the typo: DrivebaseSubsytem  ->  DrivebaseSubsystem